### PR TITLE
Pd 491 upgrade to vj6

### PR DESCRIPTION
**Pull request by Rajczakowsky**
Make sure the switch plays correct resolution/quality source by placing it in the first place in the sources array and add selected attribute to currently playing src.

### CODE REVIEW

**Comment by adamoliver** [comment]
Since this includes a breaking change to the API I think we will need to upgrade the version to 1.0.0.

File: src/plugin.js [hunk 1]
```diff
@@ -35,8 +35,39 @@ const onLanguageSelect = (player, language, selected) => {
...
+        const selectedSrc = orderedSouces.splice(index, 1);
+
+        orderedSouces.unshift(selectedSrc[0]);
+      }
+    });
+  }
+
+  player.src(language.sources.map(function(src, index) {
+    const defaultSrcData =
+    { src: src.src, type: src.type, res: src.res, label: src.label };
```
Comment (adamoliver) [inline src/plugin.js:64]: Is src.res still required?

File: src/plugin.js [hunk 2]
```diff
@@ -35,8 +35,39 @@ const onLanguageSelect = (player, language, selected) => {
...
+
+    if (!selectedCurrentSource.lenght && index === 0) {
```
Comment (adamoliver) [inline src/plugin.js]: Typo here. Should be `length`.

File: test/plugin.test.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+
+/* eslint no-console: 0 */
```
Comment (adamoliver) [inline test/plugin.test.js]: I think you should be able to remove this.

**Review by adamoliver — CHANGES_REQUESTED** [review]

File: src/plugin.js (hunk 1 above)
Comment (Rajczakowsky) [reply src/plugin.js:64]: The plugin it self don't require this attribute. It is additional information which can be used by other plugins to know what source resolution is. For example quality switch will need res to order the labels in relevance to it.

**Review by adamoliver — APPROVED** [review]

**Pull request merged**